projects/client/src/app/pages/contact/contact.tsx

```diff
@@ -0,0 +1,13 @@
+import InnerLayout from '@/app/layouts/InnerLayout';
+
+function Contact() {
+  return (
+    <InnerLayout>
+      <div className='grid h-full w-full place-content-center '>
+        <h1 className='text-3xl font-bold'>Contact Page </h1>
```

Comment:
I am not totally sure, how this works in react, but I see the same pattern on h1, p, and constant values, so... I think in react you can use const

`const pageTitle = "Contact";
`
or maybe you can use a json:

```
const jsonConstants = {
  pages: { 
    contactPage: "Contact Page"
  }
}
```

Reply:
hmmm... it is resolved?